src/sagemaker/tensorflow/estimator.py

```diff
@@ -107,7 +107,8 @@ class TensorFlow(Framework):
 
     __framework_name__ = 'tensorflow'
 
-    def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=None, py_version="py2", **kwargs):
+    def __init__(self, training_steps=None, evaluation_steps=None, checkpoint_path=None, py_version="py2",
```

Comment:
Please also update function _prepare_init_params_from_job_description in both tf and mxnet - it is used during 'attach' and we may will need to pass tag there.

Reply:
I was thinking the same @lukmis ! I am afraid that docker tag support will not work with attachment.